config/defines.inc.php

```diff
@@ -135,8 +135,10 @@
 define('_PS_TCPDF_PATH_', _PS_TOOL_DIR_.'tcpdf/');
 
 if (!defined('_PS_IMG_DIR_')) {
-    define('_PS_IMG_DIR_', _PS_ROOT_DIR_.'/img/');
+    $dir = (defined('_PS_IN_TEST_') && _PS_IN_TEST_) ? '/tests/Resources/img/' : '/img/';
+    define('_PS_IMG_DIR_', _PS_ROOT_DIR_.$dir);
```

Comment:
@matks, these changes breaks "create-test-db" command. It tries to copy demo images from wrong directory this way...
I could disable images copying in test mode, but Im not sure its good idea :sweat: 
Any suggestion?